postgres1.yml

```diff
@@ -85,7 +85,7 @@ bootstrap:
   # Some additional users users which needs to be created after initializing new cluster
   users:
     admin:
-      password: admin
+      password: admin%
```

Comment:
In your config there was a `pas%sword`, which has a `%s` placeholder, I am not sure that the `admin%` will cause the same problems.

Reply:
The error is not strictly the same but it does crash too for the same reason: 

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/patroni/postgresql/bootstrap.py", line 351, in post_bootstrap
    self.create_or_update_role(name, value.get('password'), value.get('options', []))
  File "/usr/local/lib/python3.9/dist-packages/patroni/postgresql/bootstrap.py", line 319, in create_or_update_role
    self._postgresql.query(sql)
  File "/usr/local/lib/python3.9/dist-packages/patroni/postgresql/__init__.py", line 287, in query
    return self.retry(self._query, sql, *args)
  File "/usr/local/lib/python3.9/dist-packages/patroni/utils.py", line 334, in __call__
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/patroni/postgresql/__init__.py", line 278, in _query
    raise e
  File "/usr/local/lib/python3.9/dist-packages/patroni/postgresql/__init__.py", line 267, in _query
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.9/dist-packages/psycopg/cursor.py", line 568, in execute
    raise ex.with_traceback(None)
psycopg.ProgrammingError: only '%s', '%b', '%t' are allowed as placeholders, got '%''
```